staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go

```diff
@@ -189,6 +189,7 @@ func (s *specAggregator) buildOpenAPISpec() (specToReturn *spec.Swagger, err err
 			specToReturn.Paths = nil
 			specToReturn.Definitions = nil
 		}
+		specInfo.spec.Definitions = handler.PruneDefaults(specInfo.spec.Definitions)
```

Comment:
does this mutate data held by `s.openAPISpecs`? if so, is specAggregator free to mutate that?

Reply:
Thanks, I ended-up doing a copy, and pruning a little earlier.